src/model_diagnostics/_utils/_binning.py

```diff
@@ -0,0 +1,48 @@
+import math
+
+
+def pretty(start: float, stop: float, num: int = 5) -> list:
+    r"""Pretty Intervals.
```

Comment:
```suggestion
    """Pretty Intervals.
```
No latex, no need for raw strings.